core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -83,12 +83,13 @@ object ConsumerGroupCommand extends Logging {
                 printAssignment(assignments, !opts.useOldConsumer)
               case Some("Stable") =>
                 printAssignment(assignments, !opts.useOldConsumer)
+              case None =>
```

Comment:
Sure, I'll try to do that (separate handling of `describe` for old and new consumer) later today. Thanks.